pipelines/mining/Dockerfile

```diff
@@ -19,12 +19,21 @@ RUN curl -so $HOME/google-cloud-cli.tar.gz https://dl.google.com/dl/cloudsdk/cha
  && rm $HOME/google-cloud-cli.tar.gz \
  && rm -rf ~/.cache/pip
 
-# cytominer-database and pycytominer
+# Install cytominer-database and a specific commit of pycytominr for the JUMP/CP cytomining recipe.
```

Comment:
mini nit since this is not yet merged: pycytominr --> pycytominer

Reply:
thx! typo fixed